cli/cmd/integration_gcp_pub_sub_audit.go

```diff
@@ -0,0 +1,121 @@
+//
+// Author:: David McTavish (<david.mctavish@lacework.net>)
+// Copyright:: Copyright 2023, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func createGcpPubSubAuditLogIntegration() error {
+	questions := []*survey.Question{
+		{
+			Name:     "name",
+			Prompt:   &survey.Input{Message: "Name:"},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "client_id",
+			Prompt:   &survey.Input{Message: "Client ID:"},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "private_key_id",
+			Prompt:   &survey.Input{Message: "Private Key ID:"},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "client_email",
+			Prompt:   &survey.Input{Message: "Client Email:"},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "private_key",
+			Prompt:   &survey.Editor{Message: "Enter properly formatted Private Key:"},
+			Validate: survey.Required,
+		},
+		{
+			Name: "integration_level",
+			Prompt: &survey.Select{
+				Message: "Integration Level:",
+				Options: []string{
+					api.GcpProjectIntegration.String(),
+					api.GcpOrganizationIntegration.String(),
+				},
+			},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "org_project_id",
+			Prompt:   &survey.Input{Message: "Organization/Project ID:"},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "subscription_name",
+			Prompt:   &survey.Input{Message: "Subscription Name:"},
```

Comment:
Change this to match terraform, should be Subscription ID

Reply:
👍